tests/io/test_serialize_split_modeled_energy_trace.py

```diff
@@ -22,8 +22,35 @@
     BillingElasticNetCVModel,
 )
 from eemeter.modeling.split import SplitModeledEnergyTrace
-from eemeter.testing import MockWeatherClient
-from eemeter.weather import ISDWeatherSource
+from eemeter.weather import WeatherSource
+
+
+def _fake_temps(usaf_id, start, end, normalized, use_cz2010):
+    # sinusoidal fake temperatures in degC
+    dates = pd.date_range(start, end, freq='H', tz=pytz.UTC)
+    num_years = end.year - start.year + 1
+    n = dates.shape[0]
+    avg_temp = 15
+    temp_range = 15
+    period_offset = - (2 * np.pi / 3)
+    temp_offsets = np.sin(
+        (2 * np.pi * num_years * np.arange(n) / n) + period_offset)
+    temps = avg_temp + (temp_range * temp_offsets)
+    return pd.Series(temps, index=dates, dtype=float)
+
+
+@pytest.fixture
+def monkeypatch_temperature_data(monkeypatch):
```

Comment:
Shared fixtures like this can be defined in conftest.py so that they are available everywhere.

Reply:
Moved to conftest.py